content/en/security/application_security/threats/attackers.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Attackers
+kind: documentation
+---
+
+# Understanding Attackers
+
+ASM classifies IP addresses as
+Benign
+Suspicious
+Flagged
+
+ASM additionally clusters IPs with similar attributes together.
+
+Benign: Traffic from APM that has not recently sent attack traffic
+
+Suspicious: A precursor to Flagged IPs. These IPs have met a minimum threshold of attack traffic to be classified as suspicious, but not the threshold for Flagged.
+
+Flagged: IP’s that send large amounts of attack traffic. We recommend blocking Flagged IPs.
+
+Suspicious and Flagged are available as facets. Benign traffic is shown when evaluating the impact of blocking an IP.
+
+{{< img src="security/application_security/threats/attackers/attacker_explorer_default_view.png" alt="Screenshot of the ASM Attacker Explorer"  >}}
+
+# Clusters
+{insert blurb on clusters}
+
+# Relationship to Signals and Traces
+Evidence of an attack is visible on the Trace Explorer, which identifies specific payloads that are threats.
+
+Signals correlate traces into actionable alerts.
+
+Attack Explorer identifies attackers with persistent threats which may cross multiple Signals.
```

Comment:
And then this?
```suggestion
The Attacker Explorer detects persistent threats from attackers that could span across multiple Signals.
```

Reply:
Updated based on latest draft, which removes this.